README.md

```diff
@@ -36,62 +36,35 @@ python txt_temp.py Newtem.txt output_txt_offset/
    - **Input**: `tx0_files` folder and  `GNtemp.txt` file
    - **Output**: `tx0_to_txt_temp` folder
 
-```bash
+```python
 python auto_tx0_txt_temp.py
 ```
-# Temperature Correction Formula for Resistivity
 
-The following formula is used to correct resistivity based on temperature:
-
-\[
-\rho_{\text{corrected}} = \rho_{\text{measured}} \times [1 + \alpha(T - T_{\text{ref}})]
-\]
 
+ρ_corrected = ρ_measured * [1 + α(T - T_ref)]
+
+当然,我可以为您提供这个校正公式的数学表达式。这个公式是用于根据温度校正电阻率的。让我们将其表示为数学公式:
+ρ_corrected = ρ_measured * [1 + α(T - T_ref)]
+其中:
+ρ_corrected 是校正后的电阻率
+ρ_measured 是测量的电阻率
+α 是温度系数 (在这个公式中是 0.025 /°C)
+T 是实际温度
+T_ref 是参考温度 (在这个公式中是 25°C)
+在代码中使用的具体公式是:
```

Comment:
I think remove Chinese comments and keep the english version will be better for client review.

Reply:
That is absoluely right